src/WebJobs.Script/Utility.cs

```diff
@@ -631,9 +631,16 @@ internal static bool IsSingleLanguage(IEnumerable<FunctionMetadata> functions, s
             return ContainsFunctionWithWorkerRuntime(filteredFunctions, workerRuntime);
         }
 
-        internal static string GetWorkerRuntime(IEnumerable<FunctionMetadata> functions)
+        internal static string GetWorkerRuntime(IEnumerable<FunctionMetadata> functions, IEnvironment environment = null)
         {
-            if (IsSingleLanguage(functions, null))
+            var workerRuntime = environment != null ? environment.GetEnvironmentVariable(EnvironmentSettingNames.FunctionWorkerRuntime) : null;
```

Comment:
if possible please add unit tests for this method - in https://github.com/Azure/azure-functions-host/blob/dev/test/WebJobs.Script.Tests/UtilityTests.cs

Reply:
Done.